changelog/4935.feature.rst

```diff
@@ -0,0 +1,15 @@
+Add option ``use_cls_token`` to all tokenizers. If it is set to ``True``, the token ``__CLS__`` will be added to
+the end of the list of tokens.
```

Comment:
can you add a sentence what this is used for or link to the docs? don't think this is helpfull for the average user

Reply:
this not used at all right now.
Preparing for the future

Reply:
Added a sentence to the changelog entry that this is preparation for the future and that you should not overwrite the default value for now.